packages/storybook-html/src/components/tabs/tabs.docs.mdx

```diff
@@ -0,0 +1,155 @@
+
+# Tabs
+
+<StatusBadge.Development />
+
+<a href="https://github.com/uq-its-ss/design-system/issues" class="uq-button uq-button--text" target="_blank">Check for open issues</a>
+
+## Overview
+
+The Tabs component are tabs that are aligned together which on user interaction allow navigation between groups of content that are related and at the same level of hierarchy.
+
+## Default example
+
+### Default / Outline 
+
+<Canvas withSource="none">
+  <Story id="components-tabs--tabs-outline" />
+</Canvas>
+
+```html
+  <div class="tabs tabs--outline">
+    <div class="tabs__tab-list" role="tablist" aria-label="Tab List">
+      <button class="tabs__tab tabs__tab--active" role="tab" aria-selected="true" aria-controls="tab1-panel" id="tab1">Tab 1</button>
+      <button class="tabs__tab" role="tab" aria-selected="false" aria-controls="tab2-panel" id="tab2" tabindex="-1">Tab 2</button>
+      <button class="tabs__tab" role="tab" aria-selected="false" aria-controls="tab3-panel" id="tab3" tabindex="-1">Tab 3</button>
+      <a class="tabs__link" href="#">Tab link</a>
+    </div>
+    <div class="tabs__tab-panel tabs__tab-panel--active" role="tabpanel" id="tab1-panel" aria-labelledby="tab1" tabindex="0">
+      <p>Tab1 panel text</p>
+    </div>
+    <div class="tabs__tab-panel" role="tabpanel" id="tab2-panel" aria-labelledby="tab2" tabindex="0" hidden="">
+      <p>Tab2 panel text</p>
+    </div>
+    <div class="tabs__tab-panel" role="tabpanel" id="tab3-panel" aria-labelledby="tab3" tabindex="0" hidden="">
+      <p>Tab3 panel text</p>
+    </div>
+  </div>
+```
+
+## Variant examples
+
+### Minimal
+
+<Canvas withSource="none">
+  <Story id="components-tabs--tabs-minimal" />
+</Canvas>
+
+```html
+  <div class="tabs">
+    ...
+  </div>
+```
+
+### Lined
+
+<Canvas withSource="none">
+  <Story id="components-tabs--tabs-lined" />
+</Canvas>
+
+```html
+  <div class="tabs tabs--lined">
+    ...
+  </div>
+```
+
+### With icon
+
+<Canvas withSource="none">
+  <Story id="components-tabs--tabs-with-icon" />
+</Canvas>
+
+```html
+<div class="tabs tabs--outline">
+  <div class="tabs__tab-list" role="tablist" aria-label="Tab List">
+    <button class="tabs__tab tabs__tab--active" role="tab" aria-selected="true" aria-controls="tab1-panel" id="tab1">
+        <span class="uq-icon uq-icon--search"></span> Tab 1
+    </button>
+    ...
+```
+
+### Full width
+
+<Canvas withSource="none">
+  <Story id="components-tabs--tabs-full-width" />
+</Canvas>
+
+```html
+  <div class="tabs tabs--outline tabs--full-width">
+    ...
+  </div>
+```
+
+### Small
+
+<Canvas withSource="none">
+  <Story id="components-tabs--tabs-small" />
+</Canvas>
+
+```html
+  <div class="tabs tabs--outline tabs--small">
+    ...
+  </div>
+```
+
+### Reversed
+
+<Canvas withSource="none">
+  <Story id="components-tabs--tabs-reversed" />
+</Canvas>
+
+```html
+  <div class="tabs tabs--outline tabs--reversed">
+    ...
+  </div>
+```
+
+## How and when to use
```

Comment:
These seem like generic tabs recommendations and might cover more cases than what we would like to recommend. For example, should we be telling people that they can use icon-only tabs at all? Does it fit into our UX strategy? Has this been signed-off by CX? @Bcrunk any thoughts?